kernel/src/process_utilities.rs

```diff
@@ -125,13 +151,14 @@ impl fmt::Debug for ProcessLoadError {
 /// Returns `Ok(())` if process discovery went as expected. Returns a
 /// `ProcessLoadError` if something goes wrong during TBF parsing or process
 /// creation.
-pub fn load_processes<C: Chip>(
+pub fn load_processes_advanced<C: Chip>(
```

Comment:
```suggestion
#[inline(always)]
pub fn load_processes_advanced<C: Chip>(
```

Given that load_processes has the annotation we should it include it here as well.

Reply:
I added the inline option to load_processes as it is a wrapper for load_processes_advanced. The original load_processes did not have this option.

Reply:
@hudson-ayers it's changed now, sorry I forgot to rename the variable in `process_utilities.rs`.